api/app_analytics/constants.py

```diff
@@ -1,6 +1,27 @@
+from typing import get_args
+
+from app_analytics.types import Label, PeriodType
+
 ANALYTICS_READ_BUCKET_SIZE = 15
 
 # get_usage_data() related period constants
-CURRENT_BILLING_PERIOD = "current_billing_period"
-PREVIOUS_BILLING_PERIOD = "previous_billing_period"
-NINETY_DAY_PERIOD = "90_day_period"
+CURRENT_BILLING_PERIOD: PeriodType
+PREVIOUS_BILLING_PERIOD: PeriodType
+NINETY_DAY_PERIOD: PeriodType
+(
+    CURRENT_BILLING_PERIOD,
+    PREVIOUS_BILLING_PERIOD,
+    NINETY_DAY_PERIOD,
+) = get_args(PeriodType)
+
+# Optional headers sent from client SDK mapped to their respetive labels.
+TRACK_HEADERS: dict[str, Label] = {
+    "Flagsmith-Application-Name": "client_application_name",
+    "Flagsmith-Application-Version": "client_application_version",
+}
```

Comment:
What about other default headers that we might want to track? e.g. `User-Agent` ?